content/source/docs/cloud/vcs/bitbucket-server.html.md

```diff
@@ -59,14 +59,14 @@ Leave the page open in a browser tab, and remain logged in as an admin user.
 
 1. Enter the URL of your Bitbucket Server instance in the **HTTP URL** and **API URL** fields.
```

Comment:
Would it make sense to change this to something like "Enter the URL of your Bitbucket Server instance in the **HTTP URL** and **API URL** fields excluding any trailing slashes." ?